### ISSUE TITLE
Error on Analyse Page Load

### ISSUE THREAD

**Issue by kaweepatinn1** [GoogleChrome/lighthouse#15433]
### FAQ

- [X] Yes, my issue is not about [variability](https://github.com/GoogleChrome/lighthouse/blob/main/docs/variability.md) or [throttling](https://github.com/GoogleChrome/lighthouse/blob/main/docs/throttling.md).
- [X] Yes, my issue is not about a specific accessibility audit (file with [axe-core](https://github.com/dequelabs/axe-core) instead).

### URL

nanomotions.org

### What happened?

Error on analyse page load

### What did you expect?

The analysis to happen like normal

### How were you running Lighthouse?

Chrome DevTools

### Lighthouse Version

No report generated

### Relevant log output

```shell
Mi.connect is not a function

Channel: DevTools
Initial URL: https://nanomotions.org/
Chrome Version: 116.0.0.0
Stack Trace: TypeError: Mi.connect is not a function
    at devtools://devtools/bundled/devtools-frontend/front_end/third_party/lighthouse/lighthouse-dt-bundle.js:624:662
    at Runner.gather (devtools://devtools/bundled/devtools-frontend/front_end/third_party/lighthouse/lighthouse-dt-bundle.js:527:132)
    at navigationGather (devtools://devtools/bundled/devtools-frontend/front_end/third_party/lighthouse/lighthouse-dt-bundle.js:624:548)
    at async navigation (devtools://devtools/bundled/devtools-frontend/front_end/third_party/lighthouse/lighthouse-dt-bundle.js:626:696)
    at async i (devtools://devtools/bundled/devtools-frontend/front_end/entrypoints/lighthouse_worker/lighthouse_worker.js?remoteBase=https%3A%2F%2Fchrome-devtools-frontend.appspot.com%2Fserve_file%2F%405128dafac6222fd3ac218660ab6dbf7ff260d768%2F:1:2668)
    at async self.onmessage (devtools://devtools/bundled/devtools-frontend/front_end/entrypoints/lighthouse_worker/lighthouse_worker.js?remoteBase=https%3A%2F%2Fchrome-devtools-frontend.appspot.com%2Fserve_file%2F%405128dafac6222fd3ac218660ab6dbf7ff260d768%2F:1:3056)
```
```

**Comment by NickRTR** [comment]
I'm experiencing the same issue:

```
Mi.connect is not a function

Channel: DevTools
Initial URL: http://localhost/
Chrome Version: 116.0.0.0
Stack Trace: TypeError: Mi.connect is not a function
    at devtools://devtools/bundled/third_party/lighthouse/lighthouse-dt-bundle.js:624:662
    at Runner.gather (devtools://devtools/bundled/third_party/lighthouse/lighthouse-dt-bundle.js:527:132)
    at navigationGather (devtools://devtools/bundled/third_party/lighthouse/lighthouse-dt-bundle.js:624:548)
    at async navigation (devtools://devtools/bundled/third_party/lighthouse/lighthouse-dt-bundle.js:626:696)
    at async i (devtools://devtools/bundled/entrypoints/lighthouse_worker/lighthouse_worker.js?remoteBase=https%3A%2F%2Fchrome-devtools-frontend.appspot.com%2Fserve_file%2F%40%2F:1:2683)
    at async self.onmessage (devtools://devtools/bundled/entrypoints/lighthouse_worker/lighthouse_worker.js?remoteBase=https%3A%2F%2Fchrome-devtools-frontend.appspot.com%2Fserve_file%2F%40%2F:1:3071)
```

**Comment by adamraine** [comment]
#15300 this issue should be fixed. Please check your Chrome version.

**Comment by NickRTR** [comment]
My chrome version is `116.0.5845.140` and should therefore include the fix from #15300. Additionally, the error specified in #15300 says "**N**i.connect is not a function" not "**M**i.connect is not a function" as specified in this issue.

<img width="224" alt="Bildschirmfoto 2023-09-01 um 18 53 32" src="https://github.com/GoogleChrome/lighthouse/assets/95316598/6c74d5a0-a792-41bb-897a-b4f28885f54d">
